SPECS/ca-certificates-prebuilt/ca-certificates-prebuilt.spec

```diff
@@ -0,0 +1,42 @@
+Summary:        Prebuilt version of ca-certificates package
+Name:           ca-certificates-prebuilt
```

Comment:
Suggestion: to prevent people from confusing `ca-certificates-prebuilt` with subpackages of `ca-certificates`, what do you think about changing the ordering and calling it `prebuilt-ca-certificates` or something along these lines? With the current naming I'm worried we might have people asking us why the release for `ca-certificates-prebuilt` is different from `ca-certificates` and is that bug.